src/data_generators/tests/synthetic_data_design_generator_test.py

```diff
@@ -47,6 +54,22 @@
     ],
 }
 
+TEST_LEVELS_COPULA = {
+    "largest_publisher_size": [8, 16],
+    "overlap_generator_params": [
+        GeneratorParameters(
+            "Copula",
+            CopulaDataSet,
+            {
+                "largest_pub_to_universe_ratio": 0.5,
+                "copula_generator": GaussianCopula(corr=0),
```

Comment:
Would be better if you could use values other than the default value in `CopulaDataSet`.

Reply:
Good point.  Changed.